README.md

```diff
@@ -108,6 +108,10 @@ for musl libc.
 Linux toolchain has an additional option `--enable-default-pie` to control the
 default PIE enablement for GCC, which is disable by default.
 
+To customize the enabled languages, use option `--with-languages=`. For example, 
+if you want to enable `c,c++,fortran`, use `./configure --with-languages=c,c++,fortran`.
+This option only takes effect for the GNU toolchain.
```

Comment:
"only takes effect for GCC based toolchains"?
I.e. not for LLVM?

Reply:
I think the term GNU toolchain is clear. If it turns out to be too confusing, we might adjust.

Reply:
OK - that's fine.